lobby-db/src/main/resources/db/migration/V1.09.00__lobby.sql

```diff
@@ -0,0 +1,220 @@
+-- update_tables
```

Comment:
note, section comments were added to the 1.09 and 1.10 file

These migration files will grow to be large, conflict will become something of a problem. With luck most changes will be append-only and make the conflicts easier to resolve. If things really get out of hand we can re-plan how we do future next changes and when/how to squash those changes together. Since we are small team and not so many DB changes, I think we'll manage pretty okay with the single DB migration file per release.